testproject/Assets/Tests/Manual/Scripts/PlayerMovementManager.cs

```diff
@@ -33,19 +32,22 @@ public override void OnNetworkSpawn()
 
         private void Update()
         {
-            if (NetworkObject.IsOwner && Input.GetKeyDown(KeyCode.Space))
```

Comment:
I assume this is from a different unrelated commit?

Reply:
Yeah, this was due to other PR's that changed things such that upon exiting you could end up running a NetworkBehaviour update after the NetworkObject was no longer valid.  This was so I could test the PR to make sure that the PR itself was not causing any exceptions under any edge case scenario (i.e. just getting rid of exceptions due to changes in develop branch).